### DISCUSSION TITLE
アセットのないレイアウトの微妙な問題について

### DISCUSSION THREAD

**Discussion by chaftta** [kyubuns/AkyuiUnity#33] [General]
アセットが必要としないアートボード（テキストのみ等）の場合に、空のアセットフォルダが出来てしまいます
GITだと空フォルダがコミットできないのでアートボード名.metaだけできて微妙な状態が発生してしまいます
import実行後に、後処理追加して消すなり.gitkeepを出力するなりで対応は出来なくはないのですが、スマートな解決方法が欲しいです

**Comment by kyubuns** [comment]
確かにその通りだと思ったので、v1.2.5にて自動的にgitkeepを入れてくれるTriggerを追加しました。
https://github.com/kyubuns/AkyuiUnity/releases/tag/v1.2.5

ディレクトリを消してしまうと、Triggerでディレクトリ内のファイルを舐めている処理が死ぬため
ディレクトリは必ず残した上で対応することにしました。

**Reply by chaftta** [reply]
対応ありがとうございます！
